e2e_testing/torchscript/elementwise.py

```diff
@@ -481,6 +481,43 @@ def ElementwiseRsqrtModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
 
 # ==============================================================================
+
+class ElementwiseAbsModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+
+    def forward(self, a):
+        return torch.abs(a)
+
+@register_test_case(module_factory=lambda: ElementwiseAbsModule())
+def ElementwiseAbsModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5, low=-1.0, high=0.0))
```

Comment:
nit: better to change high to 1.0 so that both negative and positive number are tested.

Reply:
yes, thank you!